File: Components/ASP.NET/VB/ASPxSchedulerDemos/App_Code/ThemeGroupModel.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections
Imports System.Collections.Generic
Imports System.Xml.Serialization
Imports System.Web.UI

Namespace DevExpress.Web.Demos

	Public Class ThemeGroupModel
		Inherits ThemeModelBase
		Private _themes As New List(Of ThemeModel)()
		Private _float As String

		<XmlElement(ElementName := "Theme")> _
		Public ReadOnly Property Themes() As List(Of ThemeModel)
			Get
				Return _themes
			End Get
		End Property

		<XmlAttribute> _
		Public Property Float() As String
			Get
				Return _float
			End Get
			Set(ByVal value As String)
				_float = value
			End Set
		End Property
	End Class

End Namespace

File: Components/ASP.NET/CS/MVCxDataEditorsDemos/Code/ThemeGroupModel.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Xml.Serialization;
using System.Web.UI;

namespace DevExpress.Web.Demos {

    public class ThemeGroupModel : ThemeModelBase {
        List<ThemeModel> _themes = new List<ThemeModel>();
        string _float;

        [XmlElement(ElementName = "Theme")]
        public List<ThemeModel> Themes {
            get { return _themes; }
        }

        [XmlAttribute]
        public string Float {
            get { return _float; }
            set { _float = value; }
        }
    }

}
